### PR TITLE
docs: add fossa badge to readme

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
Adds the fossa license check badge to the README

### CODE REVIEW

**Review by tiagolobocastro — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Mayastor Control Plane "2.0"
 
+[![FOSSA Status](https://app.fossa.com/api/projects/custom%2B39852%2Fgit%40github.com%3Aopenebs%2Fmayastor-control-plane.git.svg?type=shield&issueType=license)](https://app.fossa.com/projects/custom%2B39852%2Fgit%40github.com%3Aopenebs%2Fmayastor-control-plane.git?ref=badge_shield&issueType=license)
+
```
Comment (tiagolobocastro) [inline README.md]: While you're here, can you fix the build badge? I think the problem is the branch name is set to the old master name

Comment (Abhinandan-Purkait) [reply README.md]: Done.

Comment (Abhinandan-Purkait) [reply README.md]: Huh, but doesn't seem to work? Although if we open the link it seems to show passing.
https://mayastor-ci.mayadata.io/buildStatus/icon?job=Mayastor-Control-Plane%2Fdevelop

**Review by tiagolobocastro — CHANGES_REQUESTED** [review]
hmm it's not imported in the right project

**Review by niladrih — APPROVED** [review]

**Review by tiagolobocastro — APPROVED** [review]

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**